src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IConditionalAccessExpression.cs

```diff
@@ -300,6 +303,7 @@ void M1(P input, int? result)
             VerifyFlowGraphAndDiagnosticsForTest<BlockSyntax>(source, expectedGraph, expectedDiagnostics);
         }
 
+        [CompilerTrait(CompilerFeature.IOperation, CompilerFeature.Dataflow)]
```

Comment:
>[CompilerTrait(CompilerFeature.IOperation, CompilerFeature.Dataflow)] [](start = 8, length = 69)

We should put this attribute on the type, if it is not there already

Reply:
I'll address this in a follow up PR.

Reply:
Why include `CompilerFeature.IOperation` on the method at all? Why not just put it on the type and be done with it?